### PR TITLE
Renew token

### PR DESCRIPTION
**Pull request by bindya-esper**

### CODE REVIEW

**Review by tarun91vas — COMMENTED** [review]
Version needs to be updated.

File: docs/TokenApi.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ All URIs are relative to *https://foo-api.esper.cloud/api*
 Method | HTTP request | Description
 ------------- | ------------- | -------------
 [**get_token_info**](TokenApi.md#get_token_info) | **GET** /v1/token-info/ | Token Information
+[**renew_token**](TokenApi.md#renew_token) | **GET** /v0/enterprise/{enterprise_id}/developerapp/{developerapp_id}/renew-token/ | Renew Token
```
Comment (abdulMuneerM) [inline docs/TokenApi.md:8]: this is POST request

File: esperclient/api/token_api.py [hunk 2]
```diff
@@ -131,3 +131,114 @@ def get_token_info_with_http_info(self, **kwargs):
...
+        body_params = None
+        # HTTP header `Accept`
+        header_params['Accept'] = self.api_client.select_header_accept(
+            ['application/json'])
+
+        # Authentication setting
+        auth_settings = ['apiKey']
+
+        return self.api_client.call_api(
+            '/v0/enterprise/{enterprise_id}/developerapp/{developerapp_id}/renew-token/', 'GET',
```
Comment (abdulMuneerM) [inline esperclient/api/token_api.py:231]: request method is POST

File: docs/TokenApi.md (hunk 1 above)
Comment (bindya-esper) [reply docs/TokenApi.md:8]: Yeah. This was autogenerated as I had given it as get in the esper-spec-api. I've changed that there and raised a PR for it.

**Pull request closed without merging**